source/administration/mmctl-cli-tool.rst

```diff
@@ -51,12 +51,37 @@ the unit test coverage campaign for mmctl in the `Unit testing mmctl commands <h
 Installing mmctl
 ----------------
 
-To install the project in your `$GOPATH` run the following command as admin user:
+There are different methods available to install mmctl.
+
+**Via brew (Linux, macOS)**
+
+Use this option on Linux and macOS if you have Homebrew installed.
+
+.. code-block:: sh
+
+   brew install mmctl
+
+**Via go get (Linux, macOS, Windows)**
+
+Use this option on Linux, macOS, and Windows if you have a go environment configured.
```

Comment:
The `$GOPATH` environment variable reference belongs to this method:

```suggestion
Use this option on Linux, macOS, and Windows if you have a go environment configured. This would install the project in your `$GOPATH`:
```

Reply:
Changed it slightly differently. Let me know if it doesn't work.

Reply:
@justinegeffen just one nit, you don't need to be the admin user to install the tool, just to have the go environment set up.

Reply:
Fixed, thank you!